src/app-layer-dns-common.h

```diff
@@ -172,6 +172,8 @@ typedef struct DNSTransaction_ {
 
     TAILQ_ENTRY(DNSTransaction_) next;
     DetectEngineState *de_state;
+    uint8_t request_logged;
```

Comment:
I noticed we don't make much use of bitfields..  Is there anything wrong with:

`
    uint8_t replied:1;
    uint8_t reply_lost:1;
    uint8_t recursion_desired:1;
    uint8_t request_logged:1;
    uint8_t response_logged:1;
    uint8_t padding:3;
`